{{cookiecutter.repo_name}}/tox.ini

```diff
@@ -58,14 +57,9 @@ deps =
     -r{toxinidir}/requirements/test.txt
 commands =
     touch tests/__init__.py
-    pylint {{ cookiecutter.app_name }}
-    pylint tests
-    pylint test_utils
-    pylint --py3k {{ cookiecutter.app_name }}
-    pylint --py3k tests
-    pylint --py3k test_utils
+    pylint {{ cookiecutter.app_name }} tests test_utils
+    pylint --py3k {{ cookiecutter.app_name }} tests test_utils
```

Comment:
Digging into the pylint code, it does look like this should work, although as far as I can tell the ability to pass in multiple arguments is basically an undocumented feature.  The help output and online documentation basically don't mention the ability to pass in multiple modules or directories, except for one example of parallel execution using the `-j` parameter.  I originally did it this way because I wasn't sure it would work, and wasn't comfortable depending on undocumented behavior even if it did.